spec/models/user_spec.rb

```diff
@@ -51,4 +51,8 @@
     @user.upvoted_questions << question1
     @user.upvoted_questions << question2
   end
+
+  it "can be destroyed" do
+    expect { @user.destroy }.to_not raise_error
+  end
```

Comment:
This only tests destroying a user who has no relationships to lectures, polls, ... Add additional tests where user who are referenced by uploaded_files, lectures, courses, polls and questions are deleted.

Reply:
@WorldofJARcraft better now? I have added 3 more test cases.